spec/ruby/core/module/define_method_spec.rb

```diff
@@ -133,6 +133,25 @@ class << klass
         klass.should have_public_instance_method(:baz)
       end
     end
+
+    it "is able to navigate to super, if there is one" do
+      foo_class = Class.new do
+        def bar
+          "bar"
+        end
+      end
+
+      mixin_module = Module.new do
+        def bar
+          "super_" + super
+        end
+      end
+
+      foo = foo_class.new
+      foo.singleton_class.send(:define_method, :bar, mixin_module.instance_method(:bar))
+
+      foo.bar.should == "super_bar"
+    end
```

Comment:
**minor:** it seems a `spec/ruby/language/super_spec.rb` file is a better place for such test.

Reply:
What would be the rational for that? Checking `super` here is just a way to prove that `define_method` works as expected, hence it's secondary to the test.

Reply:
There are a lot of tests for corner cases tested in this particular file. There is a case for `define_method` as well.